src/cloud/schemas/LimitPatch.yml

```diff
@@ -0,0 +1,75 @@
+type: object
+description: These are org limits similar to those configured in/by quartz.
+properties:
+  orgID:
+    type: string
+  rate:
+    type: object
+    properties:
+      queryTime:
+        type: integer
+        description: Query Time in nanoseconds
+      readKBs:
+        type: integer
+        description: Query limit in kb/sec. 0 is unlimited.
+      concurrentReadRequests:
+        type: integer
+        description: Allowed concurrent queries. 0 is unlimited.
+      writeKBs:
+        type: integer
+        description: Write limit in kb/sec. 0 is unlimited.
+      concurrentWriteRequests:
+        type: integer
+        description: Allowed concurrent writes. 0 is unlimited.
+      cardinality:
+        type: integer
+        description: Allowed organization total cardinality. 0 is unlimited.
+      concurrentDeleteRequests:
+        type: integer
+        description: Allowed organization concurrent outstanding delete requests.
+      deleteRequestsPerSecond:
+        type: integer
+        description: Allowed organization delete request rate.
+  bucket:
+    type: object
+    properties:
+      maxBuckets:
+        type: integer
+      maxRetentionDuration:
+        type: integer
+        description: Max bucket retention duration in nanoseconds. 0 is unlimited.
+  task:
+    type: object
+    properties:
+      maxTasks:
+        type: integer
+    required: ['maxTasks']
+  dashboard:
+    type: object
+    properties:
+      maxDashboards:
+        type: integer
+    required: ['maxDashboards']
+  check:
+    type: object
+    properties:
+      maxChecks:
+        type: integer
+    required: ['maxChecks']
+  notificationRule:
+    type: object
+    properties:
+      maxNotifications:
+        type: integer
+      blockedNotificationRules:
+        type: string
+        description: comma separated list of notification rules
+        example: 'http,pagerduty'
+  notificationEndpoint:
+    type: object
+    properties:
+      blockedNotificationEndpoints:
+        type: string
+        description: comma separated list of notification endpoints
+        example: 'http,pagerduty'
+    required: ['blockNotificationEndpoints']
```

Comment:
I noticed that the `Limit.yml` schema has a [`features`](https://github.com/influxdata/openapi/blob/6ea7df4daa5735a063be3db60d0165b34b26c096/src/cloud/schemas/Limit.yml#L88) property as well. Is that intentionally not included here?

Reply:
Intentionally not included. There is no features field on idpe.Limit. The api/v2private endpoint doesn't look for that field in the request either.